app/views/whois_records/show.html.erb

```diff
@@ -0,0 +1,12 @@
+<pre>
+  <%= render partial: "whois", locals: { json: @whois_record.json } %>
+</pre>
+<% if @verified.blank? %>
+    <div class="full_info_form" style="padding: 20px 0 0 24px">
+        <%= form_for @whois_record, url: "/v1/#{@whois_record.name}", method: :get do |f| %>
```

Comment:
For all 3 comments above3: You'd have to ask Herbalizer, I converted it, since it worked, I didn't see any point in dwelling into the details of the rendered ERB.